spec/services/offender_service_spec.rb

```diff
@@ -71,6 +71,7 @@
                   offender_manager: nil,
                   service_provider: "NPS",
                   team_code: "N07UAT",
```

Comment:
As above: I don't think `team_code` is needed any more.

If we don't have any actual code using it, then we should remove it.